site/static/examples/jobs/sample-pytorchjob.yaml

```diff
@@ -14,8 +14,10 @@ spec:
         spec:
           containers:
             - name: pytorch
-              image: docker.io/kubeflowkatib/pytorch-mnist:v1beta1-45c5727
-              imagePullPolicy: Always
```

Comment:
We should use the `Always` policy. We sometimes need clarification due to installing the dedicated image.

Reply:
The image is approximately 5GB-22GB. If set to `Always`, it will pull every time, even if the local and remote image sha256 are the same. Why not consider setting it to IfNotPresent? I'm not sure if it will pull every time, even if the sha256 is the same.

Reply:
The `Always` doesn't mean that the kubelet always downloads images. Please see this for more details:

> Always
every time the kubelet launches a container, the kubelet queries the container image registry to resolve the name to an image [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier). If the kubelet has a container image with that exact digest cached locally, the kubelet uses its cached image; otherwise, the kubelet pulls the image with the resolved digest, and uses that image to launch the container.

https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy

Reply:
Thank you for clearing up my misunderstanding about 'Always.' We should indeed use it because the kubelet won't overwrite the local image if it has the same digest.

Reply:
sounds good, but you can do that in a follow up.

/lgtm
/approve